src/proxy_lib/proxy_lib.c

```diff
@@ -17,6 +17,21 @@
  * - realloc()
  */
 
+#ifdef _MSC_VER
+// Disable warning about inconsistent dll linkage:
+// warning C4273: 'malloc': inconsistent dll linkage
+// ucrt\corecrt_malloc.h(101): note: see previous definition of 'malloc'
+#pragma warning(disable : 4273)
+#pragma warning(push, 1)
+#endif
+
+#ifdef _WIN32
+// fix the error: 'malloc' redeclared without 'dllimport' attribute: 'dllexport' attribute added
+#define WIN32_dllexport __declspec(dllexport)
```

Comment:
Can't we just use the .map file like we do for other functions?

Reply:
We already use the `.def` file and this error occurs without `__declspec(dllexport)` anyway.